Student/Canvas/Calendar/UI/View/CalendarDayCell.swift

```diff
@@ -128,6 +126,10 @@ open class CalendarDayCell: UICollectionViewCell {
         dateLabel.accessibilityLabel = dateLabel.text
             .flatMap { Int($0) }
             .map(CalendarDayCell.dayOfTheMonthA11yFormatter)
+        if eventCount > 0 {
+            let format = NSLocalizedString("d_events", bundle: .core, comment: "")
+            dateLabel.accessibilityValue = String.localizedStringWithFormat(format, eventCount)
+        }
```

Comment:
Should this get updated in the case eventCount = 0?

Reply:
oh yes it should. good call good call